src/toil/jobStores/googleJobStore.py

```diff
@@ -269,6 +271,8 @@ def readFile(self, jobStoreFileID, localFilePath, symlink=False):
             with open(tmpPath, 'wb') as writeable:
                 blob = self.bucket.get_blob(compat_bytes(jobStoreFileID), encryption_key=self.sseKey)
                 blob.download_to_file(writeable)
+        if getattr(jobStoreFileID, 'executable', False):
```

Comment:
There should be an update to AbstractJobStore specifying that implementations of readFile need to respect/provide the executable attribute on FileIDs, since we're changing all the implementations.

Reply:
Should be fixed.